test/cache.js

```diff
@@ -1,21 +1,19 @@
 import test from 'ava';
 import delay from 'delay';
+import {alfy} from './fixtures/utils';
```

Comment:
Definitely better. Put it in there to make sure AVA didn't process it, forgot that `_` prevents this as well.